tests/2015-08-01/Microsoft.Compute.tests.json

```diff
@@ -242,7 +242,22 @@
       "name": "ipConfiguration - Minimal",
       "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/definitions/ipConfiguration",
       "json": {
-        "name": "ipConfiguration"
+        "name": "ipConfiguration",
+        "properties":{
+          "subnet" : {
+            "id": "resourceId" 
+          },
+          "loadBalancerBackendAddressPools": [
+            {
+              "id": "backendaddresspoolId"
+            }
+          ],
+          "loadBalancerInboundNatPools": [
```

Comment:
Is it valid for these arrays to be empty? What about having two elements in the arrays? Can you add a test for those conditions?